dashboard/lib/services/registration_reminder.rb

```diff
@@ -22,22 +22,22 @@ def self.queue_registration_reminders!
   # reminder email now.
   # @return [Enumerable<Pd::Application::ApplicationBase>]
   def self.applications_needing_first_reminder
-    # The 'registration_sent' email was sent at least two weeks ago
+    # The 'accepted_no_cost_registration' email was sent at least two weeks ago
     # No 'registration_reminder' has been sent yet.
-    # Not enrolled in a workshop since the 'registration_sent' email was sent
+    # Not enrolled in a workshop since the 'accepted_no_cost_registration' email was sent
     applications_awaiting_enrollment.
-      where("registration_sent.sent_at <= ?", 2.weeks.ago).
+      where("accepted_no_cost_registration.sent_at <= ?", 2.weeks.ago).
```

Comment:
@TurnerRiley I believe you're going to be updating the mailers for `accepted` statuses –– not sure if there will be reminder emails if they haven't enrolled, but this is the code that makes that happen.

Reply:
Gotcha, thank you!